ops/pebble.py

```diff
@@ -1314,17 +1314,7 @@ def __repr__(self):
 class NoticeType(enum.Enum):
     """Enum of notice types."""
 
-    CHANGE_UPDATE = 'change-update'
-    """Recorded whenever a change is updated, that is, when it is first
-    spawned or its status was updated. The key for change-update notices is
-    the change ID.
-    """
-
     CUSTOM = 'custom'
-    """A custom notice reported via the Pebble client API or ``pebble notify``.
-    The key and data fields are provided by the user. The key must be in
-    the format ``example.com/path`` to ensure well-namespaced notice keys.
-    """
```

Comment:
We could keep this, but I assume it's nicer to have this be an exact inverse of the PR that added the functionality.

Reply:
Yeah, I thought so too.